BUILD.md

```diff
@@ -35,25 +35,26 @@ See [System Requirements](https://xrpl.org/system-requirements.html).
 
 Building rippled generally requires git, Python, Conan, CMake, and a C++ compiler. Some guidance on setting up such a [C++ development environment can be found here](./docs/build/environment.md).
 
-- [Python 3.7](https://www.python.org/downloads/)
-- [Conan 1.60](https://conan.io/downloads.html)[^1]
-- [CMake 3.16](https://cmake.org/download/)
+- [Python 3.9](https://www.python.org/downloads/), or higher
+- [Conan 2.7.1](https://conan.io/downloads.html)[^1], or higher
```

Comment:
We required 2.17 straight ahead, just to be a bit future-proof if we will need new features.

Reply:
Thanks, I will bump this.